apis/apiextensions/v1/composition_patches.go

```diff
@@ -166,13 +166,13 @@ func (c *Patch) applyTransforms(input interface{}) (interface{}, error) {
 // patchFieldValueToObject, given a path, value and "to" object, will
 // apply the value to the "to" object at the given path, returning
 // any errors as they occur.
-func patchFieldValueToObject(fieldPath string, value interface{}, to runtime.Object) error {
+func patchFieldValueToObject(fieldPath string, value interface{}, to runtime.Object, mergeOpts *xpv1.MergeOptions) error {
```

Comment:
I'd prefer `mo` here. It's what the argument to `MergeValue` is named, and I think the type gives enough context for what they are per https://github.com/crossplane/crossplane/blob/b67e7e0/CONTRIBUTING.md#use-descriptive-variable-names-sparingly


```suggestion
func patchFieldValueToObject(fieldPath string, value interface{}, to runtime.Object, mo *xpv1.MergeOptions) error {
```